lib/measured/measurable.rb

```diff
@@ -29,11 +29,18 @@ def convert_to(new_unit)
   end
 
   def to_s
-    @to_s ||= "#{value_string} #{unit}"
+    @to_s ||= "#{value_string} #{unit.name}"
+  end
+
+  def humanize
+    @humanize ||= begin
+      unit_string = value.numerator == 1 ? unit.name : ActiveSupport::Inflector.pluralize(unit.name)
+      "#{value_string} #{unit_string}"
+    end
   end
 
   def inspect
-    @inspect ||= "#<#{self.class}: #{value_string} #{unit}>"
+    @inspect ||= "#<#{self.class}: #{value_string} #{unit.inspect}>"
```

Comment:
I think it makes sense for `#inspect` to be a "deep" operation, calling `#inspect` on all of its "children".